packages/core-ui/styles/modal.scss

```diff
@@ -1,6 +1,90 @@
 /* @license Copyright 2023 @paritytech/polkadot-dashboard-ui authors & contributors
 SPDX-License-Identifier: Apache-2.0 */
 
+.modal-content {
+  background: var(--background-modal);
+  width: 100%;
+  height: auto;
+  overflow: hidden;
+  position: relative;
+
+  h2 {
+    margin: 1rem 0;
+
+    &.unbounded {
+      font-family: Unbounded, sans-serif;
+    }
+
+    &.title {
+      font-size: 1.35rem;
+      margin: 1.25rem 0 0;
+    }
+  }
+
+  a {
+    color: var(--network-color-primary);
+  }
+
+  .header {
+    width: 100%;
+    display: flex;
+    flex-flow: row wrap;
+    align-items: center;
+    padding: 1rem 1rem 0;
+  }
+
+  .body {
+    padding: 1rem;
+  }
+
+  .action-button {
+    background: var(--button-primary-background);
+    padding: 1rem;
+    cursor: pointer;
+    margin-bottom: 1rem;
+    border-radius: 0.75rem;
+    display: flex;
+    flex-flow: row wrap;
+    align-items: center;
+    transition: all var(--transition-duration);
+    width: 100%;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+
+    h3,
+    p {
+      text-align: left;
+      margin: 0;
+    }
+
+    h3 {
+      margin-bottom: 0.5rem;
+    }
+
+    > *:last-child {
+      flex: 1;
+      display: flex;
+      flex-flow: row wrap;
```

Comment:
this can just be flex-wrap as row is default. can even test it without wrap to see if it is needed.

Reply:
actually i did test that, and without this line it works the same way the way it is now, my concern is as this component is used with the child components somewhere else outside of the same file, would it affect the child components css if they got used somewhere else when it's not under the current parent component. 
eg.: display: flex, 
         top:0;
I might be not thinking straight as I'm thinking if wrap is not default then I should keep it anyway

Reply:
Is there a scenario where the child components need a wrapped flex container? If not, then we can remove it.

Reply:
I don't think so